src/main/java/org/ideoholic/curium/model/feescategory/action/FeesAction.java

```diff
@@ -28,11 +29,12 @@ public class FeesAction {
 	private StandardActionAdapter standardActionAdapter;
 	@Autowired
 	private FeesActionAdapter feesActionAdapter;
+	@Autowired
+	private StudentActionAdapter studentActionAdapter;
 
 	@PostMapping("/applyConcession")
-	public String applyConcession() {
-		String studentId = feesActionAdapter.applyConcession();
-		return studentFeePage(studentId);
+	public String applyConcession() {;
```

Comment:
Why was the line `String studentId = feesActionAdapter.applyConcession();` removed and only a `;` was added?